stream-chat-android-ui-components/src/main/kotlin/io/getstream/chat/android/ui/mentions/MentionsListAdapter.kt

```diff
@@ -10,15 +11,22 @@ import io.getstream.chat.android.ui.databinding.StreamUiItemMentionListBinding
 import io.getstream.chat.android.ui.mentions.MentionsListAdapter.MessagePreviewViewHolder
 import io.getstream.chat.android.ui.mentions.MentionsListView.MentionSelectedListener
 import io.getstream.chat.android.ui.messagepreview.MessagePreviewView
+import io.getstream.chat.android.ui.utils.DateFormatter
 
-public class MentionsListAdapter : ListAdapter<Message, MessagePreviewViewHolder>(MessageDiffCallback) {
+public class MentionsListAdapter(
+    context: Context
+): ListAdapter<Message, MessagePreviewViewHolder>(MessageDiffCallback) {
 
     private var mentionSelectedListener: MentionSelectedListener? = null
+    private var dateFormatter = DateFormatter.from(context)
```

Comment:
Creates a single formatter for efficiency, injects it into each ViewHolder as they're being created